main.go

```diff
@@ -153,7 +170,11 @@ func main() {
 				CaAddress:      common.HexToAddress(caAddress),
 			}
 
-			rpcServer, _, _, err := rpc.InitChainServiceAndRunRpcServer(pkString, chainOpts, useDurableStore, durableStoreFolder, useNats, msgPort, rpcPort)
+			var peerSlice []string
+			if bootPeers != "" {
```

Comment:
Nitpick but I think `string.Splits` [returns an empty slice if given an empty string](https://pkg.go.dev/strings#Split), so I don't think we need the if statement

Reply:
I tried the following...
```
peerSlice := strings.Split(bootPeers, ",")
fmt.Printf("num boot peers: %d\n", len(peerSlice))
fmt.Printf("boot peers: %v\n", bootPeers)
```
... but I get the following output when I don't have any bootPeers.
```
num boot peers: 1
boot peers: 
```
I will leave for now and we can clean it up later if theres a better way